Userland/Services/LookupServer/DNSPacket.cpp

```diff
@@ -40,13 +40,14 @@ ByteBuffer DNSPacket::to_byte_buffer() const
         header.set_is_query();
     else
         header.set_is_response();
+    header.set_authoritative_answer(m_authoritative_answer);
     // FIXME: What should this be?
     header.set_opcode(0);
     header.set_response_code(m_code);
     header.set_truncated(false); // hopefully...
-    header.set_recursion_desired(true);
+    header.set_recursion_desired(false);
     // FIXME: what should the be for requests?
-    header.set_recursion_available(true);
+    header.set_recursion_available(false);
```

Comment:
Here, too: when responding to our DNS clients, we want to indicate that recursion is available (since it is).

Reply:
We're only sending DNS response packets for mDNS though - where we're not doing recursion (which seems like a more reasonable default value in general). Are you sure about this?

Reply:
I'm pretty sure [I wrote a DNS server](https://github.com/SerenityOS/serenity/pull/5345), yes :)

Reply:
Ah yes, that should be fixed as well. Sorry for that. :)